android/mfx_defs.mk

```diff
@@ -39,6 +39,12 @@ endif
 # because there is used old version of LibVA 2.0
 ifneq ($(filter MFX_O MFX_O_MR1, $(MFX_ANDROID_VERSION)),)
   MFX_CFLAGS += -DMFX_VERSION=1026
+else
+  # CPLib PAVP implementation
+  # It requires minimum API version 1.30
+  MFX_CFLAGS += \
+    -DMFX_ENABLE_CPLIB \
+    -DMFX_VERSION=1030
```

Comment:
Add MFX_ENABLE_CPLIB into https://github.com/Intel-Media-SDK/MediaSDK/blob/master/mfxconfig.h.in and make sure we get `/* #undef MFX_ENABLE_CPLIB */` line in generated mfxconfig.h.

Reply:
What is the syntax of mfxconfig.h.in? I see how to do "define" but it's not clear how to do "undef"

Reply:
You don't need to explicitly type anything for undef. cmake parses this .in file and it substitutes #cmakedefine YOUR_DEF with either  #define YOUR_DEF.... if YOUR_DEF was defined on cmake level and with /* #undef YOUR_DEF */ if it was not.

Reply:
I've added MFX_ENABLE_CPLIB into https://github.com/Intel-Media-SDK/MediaSDK/blob/master/mfxconfig.h.in and got /* #undef MFX_ENABLE_CPLIB */ line in generated mfxconfig.h. Do you want me to upload this change, or it was just a check?

Reply:
Yes, upload please. Better to have this explicit.

Reply:
done